package.json

```diff
@@ -19,9 +19,16 @@
     "css-loader": "^0.27.3",
     "file-loader": "^0.11.1",
     "rimraf": "^2.6.1",
+    "rollup": "^0.41.6",
+    "rollup-plugin-buble": "^0.15.0",
+    "rollup-plugin-gzip": "^1.0.0",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup-plugin-replace": "^1.1.1",
+    "rollup-plugin-uglify": "^1.0.1",
+    "rollup-plugin-vue": "^2.3.1",
     "uglify-js": "git://github.com/mishoo/UglifyJS2.git#harmony",
     "vue": "^2.2.6",
-    "vue-custom-element": "^1.0.13",
+    "vue-custom-element": "git+https://github.com/chimon2000/vue-custom-element.git",
```

Comment:
@kartsims I had to submit a pr to support the new rollup configuration, I will swap it out as that PR has been merged, and make the aforementioned changes.